services/server/src/models/game/game.js

```diff
@@ -43,6 +43,14 @@ const GameSchema = new Schema({
       message: '{VALUE} is not a non-negative integer value',
     },
   },
+  customURL: {
+    type: String,
+    validate: {
+      validator: (customURL) => {
+        return /^[\-0-9a-zA-Z]+$/.test(customURL) && customURL[customURL.length - 1] !== '-';
```

Comment:
do we expect customURLs to be case insensitive? I think they should be as users may copy links incorrectly (we should always convert urls to lowercase before attempting to save)

Reply:
That is a good point. We should make it case-insensitve.